parseshebang/__init__.py

```diff
@@ -45,8 +47,11 @@ def parse(file_to_parse):  # suppress(unused-function)
     """
     if hasattr(file_to_parse, "read"):
         return _parse(file_to_parse)
-    elif isinstance(file_to_parse, str):
-        path_ext = os.environ.get("PATHEXT", "").split(os.pathsep)
+    elif isinstance(file_to_parse, six.string_types):
+        if "PATHEXT" in os.environ:
+            path_ext = os.environ["PATHEXT"].split(os.pathsep)
```

Comment:
This looks fine. The only suggestion I have is that if we're going to use `frozenset` it should probably be used consistently (e.g., convert the return value of `os.environ["PATHEXT"].split(os.pathsep)` to be a `frozenset` too.